persist-cli/src/main/java/io/ballerina/persist/cmd/Init.java

```diff
@@ -167,7 +158,17 @@ public void execute() {
             }
             updateBallerinaToml(moduleName, datastore);
         } catch (BalException e) {
-            errStream.println("ERROR: failed to add database configurations in the toml file. " + e.getMessage());
+            errStream.println("ERROR: failed to add persist configurations in the toml file. " + e.getMessage());
+            return;
+        }
```

Comment:
Shall we move updating the toml file, before creating the persist directory? If persist config exists, we failed the operation but we create persist directory.

Reply:
updated in [2ebbe32](https://github.com/ballerina-platform/persist-tools/pull/174/commits/2ebbe32a5b4761a44d3f598df2e04221697ddbeb)